spec/static/openapitransactional.yaml

```diff
@@ -2379,8 +2300,8 @@ paths:
         in the link.
       operationId: getSearchSTAC
       parameters:
-      - $ref: "#/components/parameters/assetQuery"
-      - $ref: "#/components/parameters/query"
+      - #- $ref: "#/components/parameters/assetQuery"
```

Comment:
why no "-" here anymore?

Reply:
Referring to line # 2304, sorry

Reply:
this has been auto-generated ... I guess, otherwise I would have to remove it completely from the source (instead of hiding it with a comment)